### DISCUSSION TITLE
Whats the purpose of the BLE Temperature?

### DISCUSSION THREAD

**Discussion by ziogref** [revk/xESP32-Faikout#199] [General]
I just purchased 2 faikins and noticed there is an option for BLE temp sensors. I have like 9 littered around the house for Home Assistant. 

Does the faikin just display the temp in the webUI or can/does it override the Daikins built in temp sensor?

**Comment by revk** [comment]
#It has its own auto mode and can use the BLE sensor to control the room temperature.
